router/server.go

```diff
@@ -67,9 +68,19 @@ func main() {
 	tcpIP := flag.String("tcpip", "", "tcp router listen ip")
 	tcpRangeStart := flag.Int("tcp-range-start", 3000, "tcp port range start")
 	tcpRangeEnd := flag.Int("tcp-range-end", 3500, "tcp port range end")
+	certFile := flag.String("tlscert", "", "TLS (SSL) cert file in pem format")
+	keyFile := flag.String("tlskey", "", "TLS (SSL) key file in pem format")
```

Comment:
gotcha, any concerns with abusing `flag.String` to fallback to the environment variable?

```go
flag.String("tlscert", os.Getenv("TLSCERT"), "...")
```